1.cellprofiler_ic/scripts/nf1_ic.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Correct illumination and save images for each plate using CellProfiler
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+
+import utils.cp_utils as cp_utils
+
+
+# ## Set paths for each plate
+
+# In[2]:
+
+
+path_to_pipeline = pathlib.Path("NF1_illum.cppipe").absolute()
+
+plates_info_dictionary = {
+    "Plate_1": {
+        "path_to_images": pathlib.Path("../0.download_data/Plate_1/").absolute(),
+        "path_to_output": pathlib.Path("Corrected_Plate_1").absolute(),
+    },
+    "Plate_2": {
+        "path_to_images": pathlib.Path("../0.download_data/Plate_2/").absolute(),
+        "path_to_output": pathlib.Path("Corrected_Plate_2").absolute(),
```

Comment:
Are the plate numbers here arbitrary to the data itself (and instead, relative to the repo)? If there are unique plate numbers, could these be used instead? While this may be a minor point here, it may make analysis clearer when referencing multiple repositories or projects (for example, if some reference the exact same plate or differentiating when working on multiple projects at once).

Reply:
Great question! The plate numbers are the actual plate names I have for them. Due to not realizing these images already came with names (mainly cause I was new to the field), I gave them these names since it made more sense to me as Plate_1 is the pilot 96 image set and Plate_2 is the secondary pilot 384 image set. Also as you see below, these names have spaces in them so I would have had to change these, and the second plate (e.g., Cell Paint 2) had a lot of issues in it and when I got new images for this plate, the name changed also.
![image](https://user-images.githubusercontent.com/107513215/225675308-6d15b973-ef57-4225-be51-d48acd8d6b31.png)
I think for this project, giving them this naming convention is fine since the microscope seems to change the name of the plate if the images are updated.